setup.py

```diff
@@ -12,6 +12,7 @@
         "eth-typing>=2.0.0,<3.0.0",
         "eth-utils>=1.3.0b0,<2.0.0",
         "lru-dict>=1.1.6",
+        "mypy_extensions>=0.4.1,<1.0.0",
```

Comment:
Feels a little weird, like including `flake8` in the distribution. Don't feel strongly about it though.

Reply:
At the end of the day, `mypy_extensions` is a library as any other and we are using it now for things like `TypedDict`. Notice that this is not just a type hint per se. We have to have it in order for code such as the following to work.

https://github.com/ethereum/py-evm/blob/7ad4259f574db2214b03da3e8781dca5bbac04e5/eth/typing.py#L20-L25